src/Fantomas/Context.fs

```diff
@@ -405,17 +405,35 @@ let internal futureNlnCheckMem = Cache.memoizeBy (fun (f, ctx : Context) -> Cach
     let dummyCtx = ctx.With(colWriter, true)
     let writer = (dummyCtx |> f).Writer
     let str = writer.Dump()
-    let withoutStringConst = 
+    let withoutStringConst =
         str.Replace("\\\\", System.String.Empty).Replace("\\\"", System.String.Empty).Split([|'"'|])
         |> Seq.indexed |> Seq.filter (fun (i, _) -> i % 2 = 0) |> Seq.map snd |> String.concat System.String.Empty
-    let lines = withoutStringConst.Split([|Environment.NewLine|], StringSplitOptions.None) 
+    let lines = withoutStringConst.Split([|Environment.NewLine|], StringSplitOptions.None)
+
+    (lines |> Seq.length) >= 2 || writer.Column > ctx.Config.PageWidth
+
+let internal futureNlnCheckNonCached f ctx =
```

Comment:
DRY: use `futureNlnCheckNonCached` in `futureNlnCheckMem`

Reply:
Not used anymore